gslib/tests/test_stet_cp.py

```diff
@@ -48,6 +48,8 @@
 
 
 @unittest.skipIf(not system_util.IS_LINUX, 'STET binary supports only Linux.')
+# TODO: Unskip by removing the line below once stet is enabled.
+@unittest.SkipTest
```

Comment:
Isn't the syntax something like `SkipTest("Don't support X")`

Reply:
It is `skip('reason')`. Updated it.